src/main/msp/msp_serial.h

```diff
@@ -39,23 +38,24 @@ typedef enum {
 } mspEvaluateNonMspData_e;
 
 #define MSP_PORT_INBUF_SIZE 64
```

Comment:
Should we set this to more? Things like LED config could use more than 64 bytes.